site/pages/institution-en.hbs

```diff
@@ -57,7 +55,7 @@
 	</div>
 </section>
 <div class="row">
-	<div class="col-md-4 mrgn-tp-lg pull-right">
+	<div class="col-md-4 pull-right">
 		{{>most-requested}}
 	</div>
 	<section class="col-md-8 pull-left mrgn-bttm-lg">
```

Comment:
Can you do that block without the "pull-". Try to use instead "col-sm-pull-" and "col-sm-push-"